tests/tests_e3/slsa/provenance_test.py

```diff
@@ -425,6 +427,7 @@ def test_predicate_load_json() -> None:
     bd: Predicate.BuildDefinition = create_valid_build_definition()[-1]
     rd: Predicate.RunDetails = create_valid_run_details()[-1]
     predicate: Predicate = Predicate(build_definition=bd, run_details=rd)
+    sleep(2.0)
```

Comment:
I think you should add a comment to explain why this sleep is important.

Reply:
I guess I have to create an issue then, so that I can refer to it here ...

Reply:
Done (PR description updated).